uitools/toolkitcpp/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ target_include_directories(libtoolkitcpp PUBLIC
 
 target_compile_definitions(libtoolkitcpp PRIVATE
     CPP_ARCGISRUNTIME_TOOLKIT
+    MAPS_SDK_QT_TOOLKIT
```

Comment:
What is the purpose of this define (here and the pri file)? I don't see it referenced in this PR or in the toolkit, maybe I'm just not seeing it somewhere.

Reply:
I am attempting to indicate that `OAuthUserLoginPrompt` is an internal implementation detail of the Maps SDK and shouldn't be used directly in client code. In order to do that, I have this at the top of the file:
```c++
#if !defined MAPS_SDK_QT_TOOLKIT && !defined MAPS_SDK_API_BUILD
#pragma message("OAuthUserLoginPrompt.h is not intended to be used directly. API may change at any time.")
#endif
```

So the toolkit and our API build will identify themselves as known consumers, exempt from the ominous build warning.